plugins/uefi/fu-plugin-uefi.c

```diff
@@ -158,6 +158,15 @@ fu_plugin_uefi_get_splash_data (guint width, guint height, GError **error)
 	return g_bytes_new_take (g_steal_pointer (&buf), buf_idx);
 }
 
+static guint8
+fu_plugin_uefi_add_checksum (const guint8 *buf, gsize sz)
```

Comment:
This is more a "calculate" action than an "add" action to me (even if the operator is addition)

Reply:
`fu_plugin_uefi_calc_checksum` or `fu_plugin_uefi_gen_checksum` perhaps?

Reply:
Yeah either of those sound better.